docs/design/appendix_droneconfig.tex

```diff
@@ -0,0 +1 @@
+Since the content in the multirotor section is large and generous amount of information is given as to not assume the knowledge of the reader. Some of the text can be put here instead.
```

Comment:
Meant to leave this in?

Reply:
Yeah, it shouldn't show up in the main doc